Scripts/index.js

```diff
@@ -61,9 +61,9 @@ let choiceChecker = (capitalize) => {
     let humanChoice;
 
     // Using ternary operator to mimic if/else statements
-    (capitalize === "ROCK") ? humanChoice = capitalize:
-    (capitalize === "PAPER") ? humanChoice = capitalize:
-    (capitalize === "SCISSORS") ? humanChoice = capitalize:
+    (capitalize === "ROCK") ? humanChoice = "Rock":
```

Comment:
The code would output the capitalized version of the choice to the console. This was changed to output properly.